config/puma.rb

```diff
@@ -0,0 +1,16 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
+threads_count = Integer(ENV['MAX_THREADS'] || 5)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.